samples/SimpleDataSource/Tables/Metadata/FileStatsMetadataTable.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Processing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SampleCustomDataSource.Tables.Metadata
+{
+    //
+    // This is a sample Metadata table for .txt files
+    // Metadata tables are used to expose information about the data being processed, not the actual data being processed.
+    // Metadata would be something like "number of events in the file" or "file size" or any other number of things that describes the data being processed.
+    // In this sample table, we expose three columns: File Name, Line Count and Word Count.
+    //
+
+    //
+    // In order for the CustomDataSourceBase to understand your metadata table, 
+    // add a MetadataTable attribute which denotes this table as metadata.
```

Comment:
>MetadataTable [](start = 13, length = 13)

This says add a "MetadataTable" attribute, but the only attribute here is "Table"?